rdf4j/pom.xml

```diff
@@ -0,0 +1,171 @@
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.apache.commons</groupId>
+		<artifactId>commons-rdf-parent</artifactId>
+		<version>0.3.0-incubating-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>commons-rdf-rdf4j</artifactId>
+	<packaging>jar</packaging>
+	<properties>
+		<rdf4j.version>2.0</rdf4j.version>
+	</properties>
+
+	<name>Commons RDF: Integration: RDF4j</name>
+	<description>Eclipse RDF4j implementation of Commons RDF API</description>
+
+	<build>
+		<plugins>
+			<plugin>
+
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<configuration>
+					<instructions>
+						<!-- Not sure why this is needed, .impl and .internal is normally hidden 
+							by the Bundle Plugin.. -->
+
+						<Export-Package>org.apache.commons.rdf.rdf4j</Export-Package>
+						<Private-Package>org.apache.commons.rdf.rdf4j.impl</Private-Package>
+					</instructions>
+				</configuration>
+			</plugin>
+		</plugins>
+	</build>
+
+	<dependencies>
+		<dependency>
+			<groupId>${project.parent.groupId}</groupId>
+			<artifactId>commons-rdf-api</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.parent.groupId}</groupId>
+			<artifactId>commons-rdf-simple</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+
+		<!-- Selected rdf4j dependencies We don't use rdf4j-runtime or rdf4j-runtime-osgi 
+			here, as they pull in various depenencies like lucene, which might not be 
+			needed. -->
+
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-model</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-repository-api</artifactId>
+		</dependency>
+
+		<!-- For RDF4JParserBuilder, these are the RIO dependencies for the standard 
+			RDF 1.1 languages -->
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-rio-turtle</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-rio-ntriples</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-rio-nquads</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-rio-rdfxml</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-rio-trig</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-rio-jsonld</artifactId>
+		</dependency>
+		<!-- Languages notably missing, and why: 
```

Comment:
They should not need the rest of RDF4J, but can use the normal RDF4J all-in dependency if they like ; it would be compatible. I think this can go into docs.

Adding everything seemed to add multiple external libraries that might not be needed or might be in conflict with something else in a Commons RDF user's project (which after all might be doing integration).